### PR TITLE
Lesson 12

### PR DESCRIPTION
**Pull request by DariaRomanowska**

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -362,123 +470,1085 @@ input[type="submit"] {
...
+  column-gap: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  margin-bottom: 120px;
+}
+
+.reviews--top {
+  grid-row: 1 / 2;
+  position: relative;
+  margin-top: 104px;
```
Comment (5tronciy) [inline styles/styles.css]: Non-obvious place of margin declaration.  `.reviews--title{margin-bottom: 104px;}` looks clearly.

File: index.html [hunk 2]
```diff
@@ -303,13 +427,237 @@ <h2 class="title">What do we offer</h2>
...
+          <use href="#close" />
+        </svg>
+      </div>
+    </section>
+    <section class="container background-reviews">
+      <h2 class="reviews--title">
+        <span class="title-sm">Guests</span> reviews
+      </h2>
+      <div class="reviews">
+        <div class="reviews--top _bali">
```
Comment (5tronciy) [inline index.html]: Are you going to add a unique class for every possible hotel? Btw, do you use it anyway?

File: index.html [hunk 3]
```diff
@@ -303,13 +427,237 @@ <h2 class="title">What do we offer</h2>
...
+              <p>Hanna Ivanova</p>
+              <img src="./image/main/ua.png" alt="Ukraine" />
+            </div>
+          </div>
+          <div class="reviews--bottom-text">
+            <p>
+              Great location, really pleasant and clean rooms, but the thing
+              that makes this such a good place to stay are the staff. All of
+              the people are incredibly helpful and generous with their time and
+              advice.
```
Comment (5tronciy) [inline index.html:568]: Check font styles here. It differs from the design.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Looks good. Just pagination on mobile view missed.

Снимок экрана 2022-08-23 в 22 34 08

File: index.html (hunk 2 above)
Comment (DariaRomanowska) [reply index.html]: Yes, I used them to hiden for mobile version

File: index.html (hunk 3 above)
Comment (DariaRomanowska) [reply index.html:557]: Screenshot_2

it is the same as in design

**Review by 5tronciy — APPROVED** [review]
All the issues are fixed. Good work!

**Pull request merged**